docs/riff_streaming.md

```diff
@@ -8,7 +8,10 @@ title: "riff streaming"
 
 ### Synopsis
 
-<todo>
+The streaming runtime uses riff Functions, Processor and Stream custom resources to deploy streaming workloads.
+Functions can accept several input and output Streams.
+
+Message brokers and autoscaler are not provided.
```

Comment:
Autoscaler for the brokers?

Reply:
I mean: KEDA.
Both message brokers and KEDA are installed separately.
For [core](https://github.com/projectriff/cli/blob/73e04975ed0393be0443e9997cdcf60b39eb8b54/pkg/core/commands/core.go#L35), the similar section says:
```
Ingress and autoscalers are not provided.
```

Reply:
Since KEDA is a dependency of the runtime, an autoscaler is provided. Just like the knative runtime provides an autoscaler (via Knative) and ingress (via Istio).

Reply:
this sentence has been removed